### PR TITLE
FAST

### PR DESCRIPTION
**Pull request by nabobalis**
GOT TO GO FAST

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
       - run: git clone --depth 1 https://github.com/sunpy/sunpy.org.git
       - run: cd sunpy.org && git submodule update --init && cd ..
       - run: pip install -r sunpy.org/requirements.txt
-      - run: cd sunpy.org && sphinx-build -W --color -b html -d _build/.doctrees . _build/html
+      - run: cd sunpy.org && sphinx-build -j auto -W --color -b html -d _build/.doctrees . _build/html
```
Comment (Cadair) [inline .circleci/config.yml:24]: I would build one or more of these in serial so we are testing both

Comment (nabobalis) [reply .circleci/config.yml]: Done.

File: sunpy_sphinx_theme/__init__.py [hunk 2]
```diff
@@ -6,13 +6,14 @@
...
 
+import sphinx_bootstrap_theme
 
 def get_html_theme_path():
     """
     Return list of HTML theme paths.
     """
     theme_path = Path(__file__).parent.resolve() / "sunpy"
-    return theme_path
+    return [theme_path] + sphinx_bootstrap_theme.get_html_theme_path()
```
Comment (Cadair) [inline sunpy_sphinx_theme/__init__.py:16]:
```suggestion
    return [str(theme_path)] + sphinx_bootstrap_theme.get_html_theme_path()
```

Comment (Cadair) [reply sunpy_sphinx_theme/__init__.py:16]: this will break other things tho

**Pull request closed without merging**